_episodes_rmd/12-plyr.Rmd

```diff
@@ -1,5 +1,5 @@
 ---
-title: Split-Apply-Combine
+title: Splitting and Combining Data Frames with dplyr
```

Comment:
So it looks like I was wrong. This is a `plyr` lesson, not `dplyr`. Can you make that adjustment?

Reply:
Good catch! These too names are too similar!